frigg_worker/jobs.py

```diff
@@ -84,7 +84,13 @@ def succeeded(self):
     def settings(self):
         return build_settings(self.working_directory, self.docker)
 
+    def start_services(self):
+        if 'services' in self.settings:
```

Comment:
Is this necessary with https://github.com/frigg/frigg-worker/pull/44/files#diff-18b4dfb22059b19777814451cb889cfcR31?